docs/api-docs.md

```diff
@@ -2254,247 +1811,195 @@ URL: {{server_url}}/stalls/:id/upload
 Type: RAW
 URL: {{server_url}}/login
 ```
 
+**_Body:_**
 
-
-***Body:***
-
-```js        
+```js
 {
     "email": "fake_email@gmail.com",
     "password": "asdf"
 }
 ```
 
-
-
-***More example Requests/Responses:***
-
+**_More example Requests/Responses:_**
 
 ##### I. Example Request: Login user
 
+**_Body:_**
 
-
-***Body:***
-
-```js        
+```js
 {
     "email": "fake_email@gmail.com",
     "password": "asdf"
 }
 ```
 
-
-
 ##### I. Example Response: Login user
+
 ```js
 {
     "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwiaWF0IjoxNjA4Mzg4ODIwLCJleHAiOjE2MDg5OTM2MjB9.O6nzVJFmMG-bwOWWPjqMw_Ebt7pr9Csmc2A0AXeWruY"
 }
 ```
 
-
-***Status Code:*** 201
+**_Status Code:_** 201
 
 <br>
 
-
-
 ### 3. Register admin
 
-
-
-***Endpoint:***
+**_Endpoint:_**
 
 ```bash
 Method: POST
 Type: RAW
 URL: {{server_url}}/register-admin
 ```
 
+**_Body:_**
 
-
-***Body:***
-
-```js        
+```js
 {
     "email": "fake_admin_@gmail.com",
     "username": "fake_username",
     "password": "asdf"
 }
 ```
 
-
-
 ### 4. Register user
 
-
-
-***Endpoint:***
+**_Endpoint:_**
 
 ```bash
 Method: POST
 Type: RAW
 URL: {{server_url}}/register
 ```
 
+**_Body:_**
 
-
-***Body:***
-
-```js        
+```js
 {
     "email": "fake_email@gmail.com",
     "username": "fake_username",
     "password": "asdf"
 }
 ```
 
-
-
-***More example Requests/Responses:***
-
+**_More example Requests/Responses:_**
 
 ##### I. Example Request: Register user
 
+**_Body:_**
 
-
-***Body:***
-
-```js        
+```js
 {
     "email": "fake_email@gmail.com",
     "password": "asdf"
 }
 ```
 
-
-
 ##### I. Example Response: Register user
+
 ```js
 {
     "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwiaWF0IjoxNjA4Mzg3MzE2LCJleHAiOjE2MDg5OTIxMTZ9.HTTRAhau8_we-HlJsRqsJgtBcYmDFROnlLSfgKomNfQ"
 }
 ```
 
-
-***Status Code:*** 201
+**_Status Code:_** 201
 
 <br>
 
-
-
 ### 5. Update other user
 
-
-
-***Endpoint:***
+**_Endpoint:_**
 
 ```bash
 Method: PUT
 Type: RAW
 URL: {{server_url}}/users/:id
 ```
 
-
-
-***URL variables:***
+**_URL variables:_**
 
 | Key | Value | Description |
-| --- | ------|-------------|
-| id | 18 |  |
-
+| --- | ----- | ----------- |
+| id  | 18    |             |
 
+**_Body:_**
 
-***Body:***
-
-```js        
+```js
 {
     "username": "updated_fake_email@gmail.com",
     "role": "admin",
     "password": "password"
 }
 ```
 
-
-
 ### 6. Update user
 
-
-
-***Endpoint:***
+**_Endpoint:_**
 
 ```bash
 Method: PUT
 Type: RAW
 URL: {{server_url}}/updateUser
 ```
 
-
-***Headers:***
+**_Headers:_**
 
 | Key | Value | Description |
-| --- | ------|-------------|
-|  |  |  |
-
+| --- | ----- | ----------- |
+|     |       |             |
 
+**_Body:_**
 
-***Body:***
-
-```js        
+```js
 {
     "username": "hello",
     "password": "password"
 }
 ```
 
-
-
 ---
+
 [Back to top](#dk-api)
+
 > Made with &#9829; by [thedevsaddam](https://github.com/thedevsaddam) | Generated at: 2021-03-17 20:37:06 by [docgen](https://github.com/thedevsaddam/docgen)
```

Comment:
Can check what went wrong here? The markdown generated timestamp is not updated but there are so many changes to the formatting.

Reply:
ok